src/yz_events.erl

```diff
@@ -106,12 +148,24 @@ terminate(_Reason, _S) ->
 %%% Private
 %%%===================================================================
 
+%% @doc Called by {@link yz_general_sup} to create the p ETS table used to
+%% track registered events. Created when we add the handler after the supervisor
+%% is already up for yz_events.
+-spec create_table() -> ok.
+create_table() ->
+    _ = ets:new(?ETS, [named_table, public, set,
```

Comment:
@fadushin better note: I think it needs to be public to catch the fuse events. Will double check nonetheless.